### PR TITLE
Add Support to generate cell reference ballerina file ✨

### PR DESCRIPTION
**Pull request by nadundesilva**
## Automation tests
 - Unit tests 
   > N/A
 - Integration tests
   > N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: components/lang/copy-libs-linux.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```
Comment (anuruddhal) [inline components/lang/copy-libs-linux.sh]: Add License Header

File: components/lang/src/main/resources/cell_reference.bal.mustache [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Reference to the {{cellName}} cell image instance which can be used to get details about the APIs
```
Comment (anuruddhal) [inline components/lang/src/main/resources/cell_reference.bal.mustache]: Add License Header

File: components/lang/copy-libs-linux.sh (hunk 1 above)
Comment (nadundesilva) [reply components/lang/copy-libs-linux.sh]: Added. (Added to the other script as well)

File: components/lang/src/main/resources/cell_reference.bal.mustache (hunk 2 above)
Comment (nadundesilva) [reply components/lang/src/main/resources/cell_reference.bal.mustache]: Added

**Pull request merged**